build-parent/pom.xml

```diff
@@ -1007,9 +1007,6 @@ To suppress enforce-helm-version rule, you must add following plugin to the root
 
         <profile>
             <id>ci</id>
-            <properties>
-                <docker.repo.id>ghcr.io</docker.repo.id>
```

Comment:
I: Setting this here will inadvertently cause downstream projects to build against `ghcr.io` when they run the `ci` profile. Instead we should and already have configured this value in `extensions/extensions-docker/pom.xml`.